Library/Homebrew/cask/audit.rb

```diff
@@ -289,7 +289,7 @@ def audit_latest_with_auto_updates
 
     sig { params(livecheck_result: T.any(NilClass, T::Boolean, Symbol)).void }
     def audit_hosting_with_livecheck(livecheck_result: audit_livecheck_version)
-      return if cask.discontinued?
+      return if cask.deprecated? || cask.disabled?
```

Comment:
I think we also need a new (strict?) audit for usage of `caveats :discontinued`?